cmd/utils/selector.go

```diff
@@ -1,514 +0,0 @@
-	for {
-		_, err = rl.Readline()
-
-		if err != nil {
-			switch {
-			case err == readline.ErrInterrupt, err.Error() == "Interrupt":
-				err = promptui.ErrInterrupt
-			case err == io.EOF:
-				err = promptui.ErrEOF
-			}
-			break
-		}
-
-		_, idx := l.Items()
-		if idx != list.NotFound {
-			break
-		}
-
-	}
-
-	if err != nil {
-		if err.Error() == "Interrupt" {
-			err = promptui.ErrInterrupt
-		}
-		sb.Reset()
-		sb.WriteString("")
-		sb.Flush()
-		rl.Write([]byte(showCursor))
-		rl.Close()
-		return "", false, err
-	}
-
-	items, idx := l.Items()
-	item := items[idx]
-
-	sb.Reset()
-	sb.Write(render(s.OktetoTemplates.selected, item))
-	sb.Flush()
-
-	rl.Write([]byte(showCursor))
-	rl.Close()
-
-	return s.Items[l.Index()].Name, s.Items[l.Index()].IsOkteto, err
-}
-
-func (s *OktetoSelector) prepareTemplates() error {
-	tpls := s.OktetoTemplates
-	if tpls == nil {
-		tpls = &oktetoTemplates{}
-	}
-
-	tpls.FuncMap = s.Templates.FuncMap
-
-	if s.Templates.Label == "" {
-		s.Templates.Label = fmt.Sprintf("%s {{.}}: ", promptui.IconInitial)
-	}
-
-	tpl, err := template.New("").Funcs(tpls.FuncMap).Parse(s.Templates.Label)
-	if err != nil {
-		return err
-	}
-
-	tpls.label = tpl
-
-	if s.Templates.Active == "" {
-		s.Templates.Active = fmt.Sprintf("%s {{ . | underline }}", promptui.IconSelect)
-	}
-
-	tpl, err = template.New("").Funcs(tpls.FuncMap).Parse(s.Templates.Active)
-	if err != nil {
-		return err
-	}
-
-	tpls.active = tpl
-
-	if s.Templates.Inactive == "" {
-		s.Templates.Inactive = "  {{.}}"
-	}
-
-	tpl, err = template.New("").Funcs(tpls.FuncMap).Parse(s.Templates.Inactive)
-	if err != nil {
-		return err
-	}
-
-	tpls.inactive = tpl
-
-	if s.Templates.Selected == "" {
-		s.Templates.Selected = fmt.Sprintf(`{{ "%s" | green }} {{ . | faint }}`, promptui.IconGood)
-	}
-
-	tpl, err = template.New("").Funcs(tpls.FuncMap).Parse(s.Templates.Selected)
-	if err != nil {
-		return err
-	}
-	tpls.selected = tpl
-
-	if s.Templates.Details != "" {
-		tpl, err = template.New("").Funcs(tpls.FuncMap).Parse(s.Templates.Details)
-		if err != nil {
-			return err
-		}
-
-		tpls.details = tpl
-	}
-
-	if s.Templates.Help == "" {
-		s.Templates.Help = fmt.Sprintf(`{{ "Use the arrow keys to navigate:" | faint }} {{ .NextKey | faint }} ` +
-			`{{ .PrevKey | faint }} {{ .PageDownKey | faint }} {{ .PageUpKey | faint }} ` +
-			`{{ if .Search }} {{ "and" | faint }} {{ .SearchKey | faint }} {{ "toggles search" | faint }}{{ end }}`)
-	}
-
-	tpl, err = template.New("").Funcs(tpls.FuncMap).Parse(s.Templates.Help)
-	if err != nil {
-		return err
-	}
-
-	tpls.help = tpl
-
-	extraInfo := changeColorForWindows(`{{ " i " | black | bgBlue }} {{ "Use 'okteto context <URL>' to add a new cluster context" | oktetoblue }}`)
-
-	tpl, err = template.New("").Funcs(tpls.FuncMap).Parse(extraInfo)
-	if err != nil {
-		return err
-	}
-
-	tpls.extraInfo = tpl
-
-	s.OktetoTemplates = tpls
-
-	return nil
-}
-
-func (s OktetoSelector) getInitialPosition(ctx context.Context) (int, error) {
```

Comment:
this is removed, we no longer need to calculate this here, we calculate it at the context scope, also, this removes dependency with the okteto pkg